VogtPayroll5/Payroll.cs

```diff
@@ -1,34 +1,90 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Text;
+using System.IO;
 
 namespace VogtPayroll5
 {
     class Payroll
     {
+        //total number of hours worked by all employees, and total gross pay as well
+        private static int countHours;
+        private static decimal totalGrossPay;
+
         private List<Employee> _empList = new List<Employee>();
+
         public void Run()
         {
-            List<Employee> empList = new List<Employee>();
             PayrollConsoleReader payrollConsoleReader = new PayrollConsoleReader();
             Employee emp = new Employee();
             bool run = false;
 
             do
             {
+                Console.Clear();
                 emp = payrollConsoleReader.GetEmployeeInfo();
+                run = payrollConsoleReader.ReturnTrueOrFalse();
+
                 _empList.Add(emp);
             } while (run == true);
 
-            LoopThroughEmployeeInfo(emp.CalculateGrossPay(emp.HoursWorked, emp.HourlyPayRate));
+            LoopThroughEmployeeInfo(emp.CalculateGrossPay(emp.HoursWorked, emp.HourlyPayRate)+emp.CalculateOvertimePay(emp.HoursWorked, emp.HourlyPayRate));
+
+            DisplayTotalEmployeesHoursWorkedAndGrossPay();
+
+            WriteEmployeeListTotalsAndCountToFile(_empList);
+
+            File.Copy(@"C:\Users\Gabriel\source\repos\VogtPayroll5\EmployeeList.txt", @"C:\Users\Gabriel\source\repos\VogtPayroll5\EmployeeListBackup.txt");
+        }
+
+        public void DisplayTotalEmployeesHoursWorkedAndGrossPay()
+        {
+            Console.WriteLine();
+            Console.WriteLine($"Employee count: {_empList.Count}");
+            Console.WriteLine($"Employee total of hours worked {countHours}");
+            Console.WriteLine($"Employee total of gross pay {totalGrossPay:C2}");
+
         }
 
         public void LoopThroughEmployeeInfo(decimal grossPay)
```

Comment:
You're using 1 employee's `grossPay` here to calculate EVERY employee's totals? That doesn't seem right 🤔 I dont think you even need a parameter here.